models/models_file.py

```diff
@@ -0,0 +1,168 @@
+        default_keys = ["mockoon_file", "flask_debug", 'logging_level', 'host_addr']
+        logger = logging.getLogger("main")
+        for key in default_keys:
+            if key not in data.keys():
+                logger.info("Key missing: %s, loading from default", key)
+
+        return cls(mockoon_file, flask_debug, logging_level, host)
+
+    @classmethod
+    def fromFile(cls, path="config.yaml"):
+        path = abspath(path)
+        try:
+            with open(path) as f:
+                config = load(f, Loader=Loader)
+        except FileNotFoundError:
+            logger = logging.getLogger("main")
+            logger.error("Configuration file not found. Starting from defaults")
+            config = {}
+        return cls.fromDict(config)
+
+
+class Method(Enum):
+    GET = "GET"
+    POST = "POST"
+    PUT = "PUT"
+    DELETE = "DELETE"
+    PATCH = "PATCH"
+
+
+class Response:
+    def __init__(self, status_code: str, body: str, method: str, latency: int) -> None:
+        logger = logging.getLogger("main")
+        self.body = body
+        self.latency = latency
+        try:
+            self.status_code = HTTPStatus(int(status_code))
+        except ValueError as e:
+            logger.error("Your config file passed invalid status code: %s", e)
+            raise SystemExit
+        try:
+            self.method = Method(method.upper())
+        except ValueError as e:
+            logger.error("Your config file passed invalid method: %s", e)
+            raise SystemExit
+
+    def __repr__(self) -> str:
+        return f"Response {self.method} after {self.latency} {self.status_code}: {self.body}"
+
+    @classmethod
+    def fromDict(cls, data: dict, method: str) -> object:
+        body = data["body"]
+        statusCode = data["statusCode"]
+        method = method
+        latency = data["latency"]
+        return cls(statusCode, body, method, latency)
+
+
+class Endpoint:
+    def __init__(self, endpoint: str, responses: "List[Response]") -> None:
+        self.responses = responses
+        self.endpoint = "/" + endpoint
+
+    def __repr__(self) -> str:
+        return f"Endpoint: \"{self.endpoint}\" responses: {self.responses}"
+
+    @classmethod
+    def fromDict(cls, data: dict) -> object:
+        endpoint = cls.adapt_route(data['endpoint'])
+        response = data["responses"][0]
+        responses = [Response.fromDict(response, data["method"])]
+        return cls(endpoint, responses)
+
+    @classmethod
+    def responseFromDict(cls, data: dict) -> object:
+        response = data["responses"][0]
+        return Response.fromDict(response, data["method"])
+
+    @classmethod
+    def adapt_route(cls, route: str) -> str:
+        name_allowed = "A-z0-9_"
+        params_search = re.compile(f"(:[{name_allowed}]+)")
+        params = params_search.findall(route)
+        for param in params:
+            route = route.replace(param, f"<{param[1:]}>", 1)
+        return route
```

Comment:
do tego to przynajmniej słowo komentarza a najlepiej jakiś przykład użycia w komentarzu